src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java

```diff
@@ -69,6 +70,11 @@ private static void setupDrivers(Application application) {
         driver = new LineDriverAdapter(drawerController, LineFactory.getSpecialLine(), "special");
         DriverFeature.addDriver("Special line Simulator", driver);
         DriverFeature.updateDriverInfo();
+
+        DriversComposite driversComposite = new DriversComposite();
+        driversComposite.addDriver(new LineDriverAdapter(drawerController, LineFactory.getBasicLine(), "basic"));
+        driversComposite.addDriver(new LoggerDriver(true));
+        DriverFeature.addDriver("Drivers composite simulator", driversComposite);
```

Comment:
The name should specify the used drivers rather than the design pattern, e.g., BasicLine with Logger.